.github/workflows/build.yml

```diff
@@ -53,44 +47,9 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
 
-      - name: Run cargo test
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
+      - name: Install Just
+        uses: extractions/setup-just@v1
 
-  lints:
-    name: Lints
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout sources
-        uses: actions/checkout@v4
-        with:
-          submodules: true
-
-      - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          override: true
-          components: rustfmt, clippy
-
-      - uses: Swatinem/rust-cache@v2
-
-      - name: Run cargo fmt
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
-
-      - name: Run cargo clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
-          args: -- -D warnings
-
-      - name: Run rustdoc lints
-        uses: actions-rs/cargo@v1
+      - name: Run just
```

Comment:
I think this just (pun intended) needs to be:

```sh
      - name: Run just
        run: just test
```

Reply:
okay lets try that 😂

Reply:
This will lint & test on every OS in our matrix which is a bit overkill (linting should not change between OS) but oh well, its not like we're personally paying for the CI :P